css/overrides.css

```diff
@@ -347,3 +347,7 @@ span.password-toggle-wrapper {
 .cyborg .menu-dropdown a {
   color: #888888;
 }
+
+.placeholder {
```

Comment:
I'd make this more specific to messages, like `.messages .placeholder` since this might make actual placeholders a bit useless as it is now.

Reply:
done